docs/configuration.md

```diff
@@ -5,9 +5,10 @@ title: Configuration
 
 ## Environment Variables
 
-The following are environment variables used by the app:
+Honeycomb relies on environment variables during build and run time to control the different configurations. Bellow we list the  environment variables used by the app and indicate whether they are mandatory, optional and needed during build (`npm build`) or run time (using the application executable of `npm run dev`)
```

Comment:
`Bellow` => `Below`

Reply:
`relies on` is kind of strong wording since you can not set any of these and it will work in some fashion - maybe `uses`?